.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Can we get rid of this? If you're on a Mac, things like `.DS_Store` should be in your global Git config, not in each repo you work in.

Reply:
Done